gubernator/view_pr.py

```diff
@@ -68,12 +71,17 @@ def base(path):
 
 class PRHandler(view_base.BaseHandler):
     """Show a list of test runs for a PR."""
-    def get(self, pr):
-        builds = pr_builds(pr)
+    def get(self, path, pr):
+        if path:
+            repo = 'kubernetes/%s' % path.strip('/')
```

Comment:
It doesn't. I'm waiting for some examples of results from other orgs-- I think I can use the repo metadata for them.